pom.xml

```diff
@@ -607,16 +607,9 @@
           <version>3.0.0-M4</version>
         </plugin>
         <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-checkstyle-plugin</artifactId>
-          <version>3.1.2</version>
-          <dependencies>
-            <dependency>
-              <groupId>com.puppycrawl.tools</groupId>
-              <artifactId>checkstyle</artifactId>
-              <version>8.42</version>
-            </dependency>
-          </dependencies>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>2.6.1</version>
```

Comment:
nit: should we use the latest version `2.11.1` here?

Reply:
nice catch! fixed